lib/index.js

```diff
@@ -96,10 +97,9 @@ DuplicatePackageCheckerPlugin.prototype.apply = function apply(compiler) {
       if (!isSeen) {
         const entry = { version, path: modulePath };
 
-        const issuer = module.issuer && module.issuer.resource
-          ? cleanPathRelativeToContext(module.issuer.resource)
-          : null;
-        entry.issuer = issuer;
+        const issuer = isWebpack5 ? compilation.moduleGraph.getIssuer(module) : module.issuer;
+
+        entry.issuer = issuer ? cleanPathRelativeToContext(issuer.resource) : null;
```

Comment:
Should this check for issuer.resource like you were previously?
```suggestion
        entry.issuer = issuer && issuer.resource ? cleanPathRelativeToContext(issuer.resource) : null;
```

Reply:
Doesn't hurt to be safe.